ipc/pipe/pipe_factory_bench_test.go

```diff
@@ -11,37 +11,12 @@ import (
 	"go.uber.org/zap"
 )
 
-func Benchmark_WorkerPipeNoTTL(b *testing.B) {
```

Comment:
Why was this test removed?

Reply:
There were two identical tests, one with `context` and one without. Now we have only one method in factory with context.